lighthouse-core/config/default-config.js

```diff
@@ -622,6 +620,7 @@ const defaultConfig = {
         {id: 'viewport', weight: 2, group: 'pwa-optimized'},
         {id: 'apple-touch-icon', weight: 1, group: 'pwa-optimized'},
         {id: 'maskable-icon', weight: 1, group: 'pwa-optimized'},
+        {id: 'service-worker', weight: 1, group: 'pwa-optimized'},
```

Comment:
instead of being the last item in the group..
![image](https://user-images.githubusercontent.com/39191/102147598-dc8bf480-3e1f-11eb-97aa-254f0e1b48f2.png)




 lets have it be the first. bonus: it hasn't moved far from where it was.

Reply:
done :)